Services/ProfileService.cs

```diff
@@ -0,0 +1,30 @@
+using AppBackendTraining.Clients.Pms;
+using AppBackendTraining.Clients.Pms.Models.Profile;
+using AppBackendTraining.Models.Profiles;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace AppBackendTraining.Services
+{
+    public class ProfileService
+    {
+        private readonly ILogger<ProfileService> _logger;
+        private readonly PmsService _pmsService;
+        public ProfileService(ILogger<ProfileService> logger, PmsService pmsService)
+        {
+            _logger = logger;
+            _pmsService = pmsService;
+        }
+
+        public async Task CreateProfile(ProfileCreateRequest profileCreateRequest, long accountId)
+        {
+            await _pmsService.CreateProfile(
```

Comment:
Try to avoid await when not consuming result locally. Simply return the task and let the caller await. This avoid creating useless continuation, reduces contention.